runner/src/main/scala/stress/utils/Scenarios.scala

```diff
@@ -104,6 +104,23 @@ object Scenarios {
       )
   }
 
+  def bulkOpsScenario(repeats: Int): ScenarioBuilder = {
+    scenario("(Bulk) Ops person")
+      .repeat(repeats)(
+        exec(Releases.getRandomTreeRelease)
```

Comment:
Not a 100% certain but this means that you depend on having dependency trees generated, which are used for the dependencies scenario? So without reading the source code I have no idea that the two are correlated, so maybe update the read me or reuse the active releases (which are generated by default) or something...